### PR TITLE
Improve testing and error message for IPv6 rejeciton

### PR DESCRIPTION
**Pull request by marc-casperlabs**
This adds a test and a better error in case of rejection.

### CODE REVIEW

File: node/src/utils.rs [hunk 1]
```diff
@@ -607,4 +609,10 @@ mod tests {
 
         assert_eq!(extracted, expected);
     }
+
+    #[test]
+    fn resolve_address_rejects_ipv6() {
```
Comment (rafal-ch) [inline node/src/utils.rs:614]: Not a blocker, but since I'm paranoid, I'd consider adding:
```
fn resolve_address_accepts_ipv4()
```

**Review by rafal-ch — APPROVED** [review]
LGTM :+1:

**Review by Fraser999 — APPROVED** [review]

File: node/src/utils.rs (hunk 1 above)
Comment (marc-casperlabs) [reply node/src/utils.rs:614]: -.-

Comment (marc-casperlabs) [reply node/src/utils.rs:614]: See 4aa04b746

**Pull request merged**